### PR TITLE
[v1.13.x] Bump Alpine version in Kubectl Docker image to resolve CVEs

### PR DESCRIPTION
**Pull request by ben-taussig-solo**
# Description
 - Bump Alpine version in Kubectl Docker image to resolve [CVE-2023-0286](https://avd.aquasec.com/nvd/2023/cve-2023-0286/)
 - Use the following steps to validate these changes:
   - Build the Kubectl Docker image locally after checking out this branch
     - `TAGGED_VERSION=v1.14.0-beta10-kubectl-bump make kubectl-docker`
   - Scan the built image, inspect the output in `_output/scans`, and confirm that there are no CVEs detected
     - `TAGGED_VERSION=v1.14.0-beta10-kubectl-bump make scan-version`

### CODE REVIEW

File: jobs/kubectl/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM bitnami/kubectl:1.24.7 as kubectl
 
-FROM alpine:3.15.6
+FROM alpine:3.17.2
```
Comment (jenshu) [inline jobs/kubectl/Dockerfile:3]: why is this the only one affected? don't all our other images on 3.15.6 need to be updated as well?

Comment (ben-taussig-solo) [reply jobs/kubectl/Dockerfile:3]: Everything else is already on 3.17

Comment (ben-taussig-solo) [reply jobs/kubectl/Dockerfile:3]: Looks like that update was done as part of this PR https://github.com/solo-io/solo-projects/pull/4426
It seems like we missed kubectl, but I'll reach out to Sai and Jake to confirm that there isn't a specific reason we didn't bump the alpine version for Kubectl

Comment (jackstine) [reply jobs/kubectl/Dockerfile:3]: Great find!

Comment (ben-taussig-solo) [reply jobs/kubectl/Dockerfile:3]: my mistake -- accidentally linked a solo-projects PR

Comment (jackstine) [reply jobs/kubectl/Dockerfile:3]: We might change all the images in Gloo.  We updated the images in Solo-Projects already.

Comment (ben-taussig-solo) [reply jobs/kubectl/Dockerfile:3]: Bumped all Alpine base images to 3.17.2 here: https://github.com/solo-io/gloo/pull/7848/commits/a6e24d2120e49e2d7b6c070a1bd7b4299539e118

**Review by jenshu — APPROVED** [review]

**Pull request merged**